src/updateUI.js

```diff
@@ -0,0 +1,95 @@
+// Use DevCycle variables to control the UI 
+export const updateUI = (devcycleClient) => {
+    const demoFeature = devcycleClient.allFeatures()['hello-togglebot'];
+
+    updateToggleBotSection(devcycleClient, demoFeature);
+    updateInstructionsSection(demoFeature);
+}
+
+const updateToggleBotSection = (devcycleClient, demoFeature) => {
+    // Display variation name
+    document.getElementById("variation-name").innerHTML = `"${demoFeature?.variationName ?? 'Default'}"`;
+
+    // Get variables from DevCycle
+    const shouldWink = devcycleClient.variableValue('togglebot-wink', false);
+    const spinSpeed = devcycleClient.variableValue('togglebot-speed', 'off');
+
+    // Display message based on spinSpeed variable
+    const greeting =
+        spinSpeed === 'surprise' ? 'What the unicorn?' :
+            spinSpeed === 'off-axis' ? "...I’m gonna be sick..." :
+                spinSpeed === 'fast' ? "This is fun!" :
+                    spinSpeed === 'slow' ? "Awesome, look at you go!" :
+                        "Hello! Nice to meet you.";
```

Comment:
this many nested ternaries can be hard to read, what about a helper method with a switch statement?

Reply:
good idea, fixed!